projects/aca-shared/store/src/public-api.ts

```diff
@@ -53,3 +53,5 @@ export * from './selectors/app.selectors';
 export * from './states/app.state';
 
 export * from './store.module';
+
+export * from './testing/initial-app-state.mock';
```

Comment:
we don't export mock otherwise will became part of the bundle

Reply:
Fixed